starter_code/views/page-movie.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="page">
+    <div class="movie_image">
+        <img src="{{movie.image}}" alt="">
+    </div>
+    <div class="movie_text">
+        <a href="/movies">Go back</a>
+        <h2>{{movie.title}}</h2>
+        <h3>Director: {{movie.director}}</h3>
+        <h4>Stars</h4>
+        <ul class="movie_stars">
+            {{movie.stars}}
+        </ul>
```

Comment:
The bullet point isn't a css trick, in order to have the bullet point in a ul, you need to have "li" elements inside.
SInce the movie.stars key is an array, you need to loop over it in order to create an "li" element for each movie stars.

        `<ul class="movie_stars">`
            {{#each movie.stars}}
            <li>
                {{this}}
            </li>
            {{/each}}
        </ul>